app/models/ShortLinkRepository.scala

```diff
@@ -16,6 +16,10 @@ class ShortLinkRepository @Inject() (dbConfigProvider: DatabaseConfigProvider)(i
   import dbConfig._
   import profile.api._
 
+  // TODO -
+  //  Ideally, these shortlinks would have a randomly generated ID (bigint/UUID)
+  //  I bumped into some Slick errors while trying to set up Slick so I kept the
+  //  ID as is for now.
   private class ShortLinkTable(tag: Tag) extends Table[ShortLink](tag, "short_link") {
     def id = column[Long]("id", O.PrimaryKey, O.AutoInc)
```

Comment:
I created a new `ShortLinkDto` object to not expose the ID (and the boolean `expired` flag) from the client, but currently the shortlinks can still be easily enumerated from the database.